api/v1beta1/common.go

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 package v1beta1
 
 import (
-	corev1 "k8s.io/api/core/v1"
 	"github.com/openstack-k8s-operators/lib-common/modules/common/endpoint"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Does this shift makes any impact? (me not aware about import positioning)

Reply:
No it's not, the order is managed by the golang linter: I usually run `pre-commit` checks (defined in the repo) that takes care about this kind of things.